src/mbgl/geometry/buffer.hpp

```diff
@@ -36,11 +37,11 @@ class Buffer : private util::noncopyable {
 
     // Transfers this buffer to the GPU and binds the buffer to the GL context.
     void bind(gl::ObjectStore& store) {
-        if (buffer.created()) {
-            MBGL_CHECK_ERROR(glBindBuffer(bufferType, getID()));
+        if (buffer) {
+            MBGL_CHECK_ERROR(glBindBuffer(bufferType, (*buffer).get()));
```

Comment:
I'd kept `.get()` to keep in sync with the other `Unique*` usage elsewhere, but your suggestion looks better, thanks!